cmd/pebble/cmd_boot_firmware_common_test.go

```diff
@@ -0,0 +1,48 @@
+//go:build !termus
+// +build !termus
```

Comment:
Okay, I caught up with Gustavo about this, and he said he'd recommended a build tag a while ago, because of all the things that may be added to `boot-firmware` in future (potentially drivers and so on). That's fair enough.

Reply:
I say let's adopt this early and guide the architecture towards build tags, since other teams might also require it soon enough. I'm also not a fan of a pebble binary having logic that's essentially dead code for many targets, especially considering that Termus logic is particularly specific and irrelevant for the common Pebble use case.